airbrake/notice.py

```diff
@@ -48,20 +48,19 @@ def __init__(self, exception, params=None, session=None,
 
         self.notifier = notifier
 
-        if isinstance(exception, basestring):
+        if isinstance(exception, Exception) or \
+                isinstance(exception, BaseException):
```

Comment:
You can check for multiple types with one call to `isinstance` by passing it a tuple of types to check:

```python
if isinstance(exception, (Exception, BaseException):
```

However, this shouldn't be necessary since `Exception` is a subclass of `BaseException` so the following should suffice:

```
if isinstance(exception, BaseException):
```

No harm to use the first suggestion (with the tuple) though, just FYI. 

I think in everything beyond 2.6 it is a requirement that in order to  be `raise`able it must inherit from `BaseException`.